RevenueCatUI/Templates/V2/Components/Stack/StackComponentViewModel.swift

```diff
@@ -195,7 +195,7 @@ struct StackComponentStyle {
             case .scroll:
                 return true
             }
-        })
+        }) ?? false
```

Comment:
This breaks the vertical stack.

| Can't Scroll Now | How it worked before |
|:--:|:--:|
|![one](https://github.com/user-attachments/assets/0c80d0ca-29b3-427c-b22f-5654440129d7)|![two](https://github.com/user-attachments/assets/d551ed4b-f690-40ad-8a6b-c3e37b2d8cc6)|

In the left video, the sheet gets pulled around but I can't scroll to see the content off the bottom

Reply:
yeah, let me revert this change, because I was trying to fix something else, the fact that horizontal stacks are scrollable by default if they overflow, and it looks like that's uncovering other issues and broke vertical scroll